arbitrum-docs/build-decentralized-apps/how-tos/migrate-nfts-arbitrum.mdx

```diff
@@ -0,0 +1,137 @@
+---
+id: migrate-nfts-arbitrum
+title: "How to migrate your NFT collection to Arbitrum"
+sidebar_label: "Migrate your NFT collection to Arbitrum"
+description: Learn how to migrate your NFT collection to Arbitrum through third-party bridges and DIY bridging solutions.
+author: amarrazza
+content-type: how-to
+reader_audience: Developers who want to migrate their NFT collections to Arbitrum from either Ethereum mainnet or other non-EVM chains.
+---
+
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is for **developers** who want to migrate existing <a data-quicklook-from="non-fungible-tokens-nfts">NFT</a> collections to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a>.
+
+This guidance applies to NFT collections used within web3 games, decentralized apps (dApps), and other use-cases.
+
+### Prerequisites
+
+ - An NFT collection that's minted and managed by smart contracts on either an **EVM-compatible** chain (like Ethereum mainnet) or a supported **non-EVM** chain[^1].
+ - NFTs that are implemented using the ERC-721 or ERC-1155 token standards[^2]. (todo - clarify how this prereq can be satisfied for non-EVM chains)
+
+
+:::info
+
+ERC-721 and ERC-1155 tokens **cannot be migrated to Arbitrum via [bridge.arbitrum.io](https://bridge.arbitrum.io/)**. The Arbitrum Bridge only supports ERC-20 tokens.
+
+For guidance on how to migrate ERC-20 tokens to Arbitrum, see [How to bridge ERC-20 tokens to Arbitrum](#todo) instead.
+
+:::
+
+
+## Why should I migrate my project's NFTs to Arbitrum?
+
+<!-- todo: this can be moved to a dedicated concept doc in the future -->
+
+There are three main reasons to migrate your your project's NFTs to Arbitrum:
+
+ 1. **Low fees**. NFTs hosted by one of Arbitrum's chains can be minted, transfered, and traded at a much lower cost compared to Ethereum mainnet.
+ 2. **Fast transactions**. From the end-user's perspective, transactions on Arbitrum's chains are processed more quickly than they are on Ethereum mainnet. This lets you build NFT-powered experiences that feel just about as fast as traditional apps and games.
+ 3. **Interoperability with the Ethereum and Arbitrum ecosystems**. If you're migrating your project's NFTs from a non-EVM chain, migration to Arbitrum will let your users freely roam between Ethereum dApps, carrying their NFTs along with them. More precisely, this means that your users' NFTs will be usable across dApp boundaries, as long as those dApps' smart contracts are hosted by EVM-compatible chains.
+
+<!-- todo: when shouldn't you? Risks to call out? -->
+
+
+## How do I migrate my project's NFTs to Arbitrum?
```

Comment:
There's one more option that might be useful for devs, although I haven't seen any actual implementation of it (I think some projects have done it though) => Build your contract on Arbitrum and have your data in another chain, or the other way around. Devs might want to have Ethereum-security for the data of the NFTs, and lower-fees and higher speed for transacting with them (for example, for brand NFTs with data that is not going to change often). Or the other way around, lower-fees and higher speed to handle data, and Ethereum-security for the NFTs (for example, for gaming NFTs with data that will change often, when playing the game, or updating items).

It would be better to first find an implementation of this approach though, in case we prefer to avoid this complex scenario.